docs/manuals/characterization/t2hahn.rst

```diff
@@ -125,7 +125,7 @@ computed for other qubits.
 
 .. jupyter-execute::
 
-    exp_with_p0 = T2Hahn(physical_qubits=[qubit], delays=delays, num_echoes=number_of_echoes)
+    exp_with_p0 = T2Hahn(physical_qubits=(qubit,), delays=delays, num_echoes=number_of_echoes)
```

Comment:
Some example code uses list and others use tuple for physical_qubits. Do we want to unify them?

Reply:
Good point, updated in [18eff9c](https://github.com/Qiskit/qiskit-experiments/pull/1186/commits/18eff9c3dcb88663a8b2c5dcfa894d838add0860).